src/worldofzuul/Trump.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package worldofzuul;
+
+/**
+ *
+ * @author Kasper
+ */
+public class Trump extends Character {
+    String name;
```

Comment:
Add docblock for attribute

Reply:
the attribute should be private.

Reply:
Fixed both in 2c64b148ac86de64b48b13ee22f6b153531114b3